src/common.speech/IntentServiceRecognizer.ts

```diff
@@ -178,41 +178,45 @@ export class IntentServiceRecognizer extends ServiceRecognizerBase {
                 // If LUIS didn't return anything, send the existing event, else
                 // modify it to show the match.
                 // See if the intent found is in the list of intents asked for.
-                let addedIntent: AddedLmIntent = this.privAddedLmIntents[intentResponse.topScoringIntent.intent];
 
-                if (this.privUmbrellaIntent !== undefined) {
-                    addedIntent = this.privUmbrellaIntent;
-                }
-
-                if (null !== intentResponse && addedIntent !== undefined) {
-                    const intentId = addedIntent.intentName === undefined ? intentResponse.topScoringIntent.intent : addedIntent.intentName;
-                    let reason = ev.result.reason;
+                if (null !== intentResponse && !!intentResponse.topScoringIntent && !!intentResponse.topScoringIntent.intent) {
```

Comment:
Would it be tough to add a test case for this? "No recognized events with continuous intent" seems like a nicely defined condition we could have a case for, and this logic (which looks right to me) is non-trivial enough that guarding it against future regressions seems smart.

Reply:
Shouldn't be too hard, I'll get started on that now